File: app/src/main/java/com/example/mentorsjoy/PdfData.java
package com.example.mentorsjoy;
import java.io.Serializable;
public class PdfData implements Serializable{
    String mentorTitle; //Дожность научрука
    String mentorName; //Имя научрука
    String academicalTitle; //Должность руководителя обр. программы
    String academicalName; //Имя руководителя обр. программы
    String projectName; //Название проекта
    String projectType; //Тип проекта (числа из кодификатора)
    String studentName; //Имя студента
    String studentGroup; //Группа студента
    public void setMentorTitle(String s) {mentorTitle = s;}
    public String getMentorTitle() {return mentorTitle;}
    public void setMentorName(String s) {mentorName = s;}
    public String getMentorName() {return mentorName;}
    public void setAcademicalTitle(String s) {academicalTitle = s;}
    public String getAcademicalTitle() {return academicalTitle;}
    public void setAcademicalName(String s) {academicalName = s;}
    public String getAcademicalName() {return academicalName;}
    public void setProjectName(String s) {projectName = s;}
    public String getProjectName() {return projectName;}
    public void setProjectType(String s) {projectType = s;}
    public String getProjectType() {return projectType;}
    public void setStudentName(String s) {studentName = s;}
    public String getStudentName() {return studentName;}
    public void setStudentGroup(String s) {studentGroup = s;}
    public String getStudentGroup() {return studentGroup;}
    public PdfData() {
        mentorTitle = "aboba";
        mentorName = "aboba";
        academicalTitle = "aboba";
        academicalName = "aboba";
        projectName = "aboba";
        projectType = "aboba";
        studentGroup = "aboba";
        studentName = "aboba";
    }
}

File: app/src/main/java/com/example/mentorsjoy/PdfData.kt
package com.example.mentorsjoy

data class PdfData(val _projectName: String) : java.io.Serializable{
    private var mentorTitle //Дожность научрука
            : String? = null
    private var mentorName //Имя научрука
            : String? = null
    private var academicalTitle //Должность руководителя обр. программы
            : String? = null
    private var academicalName //Имя руководителя обр. программы
            : String? = null
    private var projectName //Название проекта
            : String? = null
    private var projectType //Тип проекта (числа из кодификатора)
            : String? = null
    private var studentName //Имя студента
            : String? = null
    private var studentGroup //Группа студента
            : String? = null

    fun setMentorTitle(s: String?) {
        mentorTitle = s
    }

    fun getMentorTitle(): String? {
        return mentorTitle
    }

    fun setMentorName(s: String?) {
        mentorName = s
    }

    fun getMentorName(): String? {
        return mentorName
    }

    fun setAcademicalTitle(s: String?) {
        academicalTitle = s
    }

    fun getAcademicalTitle(): String? {
        return academicalTitle
    }

    fun setAcademicalName(s: String?) {
        academicalName = s
    }

    fun getAcademicalName(): String? {
        return academicalName
    }

    fun setProjectName(s: String?) {
        projectName = s
    }

    fun getProjectName(): String? {
        return projectName
    }

    fun setProjectType(s: String?) {
        projectType = s
    }

    fun getProjectType(): String? {
        return projectType
    }

    fun setStudentName(s: String?) {
        studentName = s
    }

    fun getStudentName(): String? {
        return studentName
    }

    fun setStudentGroup(s: String?) {
        studentGroup = s
    }

    fun getStudentGroup(): String? {
        return studentGroup
    }

    init{
        projectName = _projectName
    }
}
